.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,27 @@
+---
+name: Feature Proposal
+about: Propose a new feature
+---
+
+<!--
+    ⚠️ Review existing issues to see whether someone else has already reported your issue.
+
+    📖 Please also read CONTRIBUTING.md.
+-->
+
+## Feature description
+
+<!--
+    Think through your proposal and describe it clearly.
+
+    Note that features are only added to the most recent version of Git Extensions.
+-->
+
+
+
+
```

Comment:
I suggest adding Environment here too, so we can see if a user have attempted to use any later version

Reply:
Good idea. Fixed.